packages/components/src/Button/Button/_docs/Button.stickersheet.stories.tsx

```diff
@@ -280,10 +280,10 @@ export const StickerSheetRTL: StickerSheetStory = {
   ...StickerSheetTemplate,
   name: "Sticker Sheet (RTL)",
   parameters: {
-    ...StickerSheetTemplate.parameters,
     chromatic: {
       delay: 1200,
     },
+    ...StickerSheetTemplate.parameters,
```

Comment:
Oh? Strange change. Doesn't make a difference in here given the attribute isn't duplicated, but I think spreads should go above unless you want to prevent overriding the value 🤔